packages/cli/src/lib/cmds/play.ts

```diff
@@ -0,0 +1,123 @@
+import {Prompt, InquirerPrompt} from '../Prompt';
+
+class Play {
+  constructor(
+    private args: ParsedArgs,
+    private config: Config,
+    private androidSdkTools: AndroidSdkTools,
+    private googlePlay: GooglePlay,
+  ){}
+
+  async bootstrapPlay(): Promise<void> {
+    await this.googlePlay.initPlay();
+  }
+
+  async getLargestVersion(): Promise<void> { 
+    // TODO(nohe427): This doesn't exist in the Gradle Play Plugin. Might be wortwhile to build
+      // small integration to do this by getting the service account file?
+  }
+
+  private isInAvailableTracks(userSpecifiedTrack: string): Boolean {
+    const track: string = (userSpecifiedTrack).toLowerCase();
+    const selectedTrack: Track = (<any>Track)[track];
+    if(selectedTrack == (null || undefined)) {
+      return false; // Should probably spit out an error message that track needs to be in [x,y,z]
+    }
+    return true;
+  }
+
+  // bubblewrap play --publish="Internal"
+  async publish(): Promise<void> {
+    //if (this.args.publish) { // This argument should be validated in run()
+    //}
+    //Validate that the publish value is listed in the available Tracks.
+    const userSelectedTrack = this.args.publish.toLowerCase() || "internal"; // If no value was supplied with publish we make it internal.
+    if(!this.isInAvailableTracks(userSelectedTrack)) {
+      return; // Throw error message?
+    }
+    if (this.args.appBundleLocation) {
+      // Check this is a directory that contains our specified file name.
+    }
+
+    
+      // Make tmp directory copy file over signed APK then cleanup.
+      // await this.googlePlay.publishBundle(Track.Internal,)
+  }
+
+  private validServiceAccountJsonFile(path: string | undefined): Boolean { // Return an error or boolean? Log a message?
+    if(path == undefined) {
+      // Log an error
+      return false;
+    }
+    if (!fs.existsSync(path)) {
+      // path doesn't exist log an error
+      return false; 
+    }
+    return true;
+  }
+
+  // bubblewrap play --init
+  async run(): Promise<boolean> {
+    const manifestFile = this.args.manifest || path.join(process.cwd(), TWA_MANIFEST_FILE_NAME);
+    const twaManifest = await TwaManifest.fromFile(manifestFile);
+    // Update the TWA-Manifest if service account is supplied
+    // bubblewrap play --serviceAccountFile="/path/to/service-account.json"  --manifest="/path/twa-manifest.json"
+    if (this.args.serviceAccountFile) {
+      // Add service account to the TWA-Manifest
+      const serviceAccountFile = this.args.serviceAccount;
+      twaManifest.serviceAccountJsonFile = serviceAccountFile;
+      // Then we need to call bubblewrap update so the gradle plugin has the appropriate file.
+    }
+    // Check if service account file exists on disk.
+
+    // Need to validate that the service account file exists in TWA-Manifest
+    // and/or on disk. (Thinking about CI/CD scenarios)
+    if (this.validServiceAccountJsonFile(twaManifest.serviceAccountJsonFile)) {
```

Comment:
I think there should be a "not" in this line.

Reply:
Correct. Added that in